primitives/state-machine/src/in_memory_backend.rs

```diff
@@ -364,6 +364,10 @@ impl<H: Hasher> Backend<H> for InMemory<H> where H::Out: Codec {
 	fn usage_info(&self) -> UsageInfo {
 		UsageInfo::empty()
 	}
+
+	fn wipe(&self) -> Result<(), Self::Error> {
+		Ok(())
+	}
```

Comment:
This seems to wipe the in memory backend just like I want... but why? All I did was return `Ok(())`

Reply:
To be clear, everything works, I just want to know why...